CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#7870](https://github.com/apache/trafficcontrol/pull/7870) *Traffic Portal*: Adds a hyperlink to the DSR page to the DS itself for ease of navigation.
 
 ### Changed
+- [#62] (https://github.com/comcast-cdn/trafficcontrol/pull/62) *Traffic Monitor, Documentation* Removed anycast functionality from traffic control
```

Comment:
There is no ATC release containing Anycast. So instead of adding an additional changelog entry for removing it, let's just remove the original anycast changelog entry.